priv/repo/migrations/20210921025336_change_image_scale_option_type.exs

```diff
@@ -0,0 +1,20 @@
+defmodule Philomena.Repo.Migrations.ChangeImageScaleOptionType do
+  use Ecto.Migration
+  import Ecto.Query
+
+  def change do
+    alter table(:users) do
+      add :scale_large_images0, :binary, default: "true", null: false
+    end
+  
+    flush() # Not sure if this is necessary
+  
+    execute("update users set scale_large_images0 = (CASE WHEN scale_large_images THEN 'true'::bytea ELSE 'false'::bytea END) where scale_large_images is not null;")
```

Comment:
```sql
update users set scale_large_images0 = (case
when scale_large_images then 'true'
else 'false'
);
```
`scale_large_images` already can't be null because it has a nullability constraint

Reply:
Ah, right; thanks